### PR TITLE
feat: update to follow the new branching strategy

### PR DESCRIPTION
**Pull request by inghylt**
**Related issue(s) and PR(s)**  
This PR relates to #963.

**Type of change**  

- [x] Other, updating documentation
 
**List of changes made**  

- Updated CONTRIBUTING.md

**Testing**  

Please read through the changes and see if they look good

**Definition of Done checklist**  
- [x] My code meets the Acceptance Criteria
- [x] My code follows the [NBIS style guidelines](https://github.com/NBISweden/development-guidelines)
- [x] I have performed a self-review of my own code and commented any hard-to-understand areas
- [x] Tests and lint/format validations are passing
- [x] My changes generate no new warnings

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ If possible, please also add Acceptance Criteria so it is clear what is required
 
 ## Making Changes
 
-The production website, https://metabolicatlas.org, is based on the `main` branch. For development, new work should be based on the `develop` branch.
+The production website, https://metabolicatlas.org, is based on the `main` branch. All new development should be based on this branch.
 
 ### Getting Started
 
 To get started with running the project on your local development machine, please follow the [README](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/README.md)
```
Comment (e0) [inline CONTRIBUTING.md:23]:
```suggestion
To get started with running the project on your local development machine, please follow the [README](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/main/README.md)
```

Comment (e0) [reply CONTRIBUTING.md:23]: Similarly, the link in the `README` should also be changed. The last sentence in the screenshot could probably be removed.

image

**Review by e0 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (inghylt) [reply CONTRIBUTING.md]: Nice catch, sloppy that I did not find it myself. Fixed in 4271d4256c54d1b3b0ca3b818322ecbd350cb399

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,15 +16,15 @@ If possible, please also add Acceptance Criteria so it is clear what is required
 
 ## Making Changes
 
-The production website, https://metabolicatlas.org, is based on the `main` branch. For development, new work should be based on the `develop` branch.
+The production website, https://metabolicatlas.org, is based on the `main` branch. All new development should be based on this branch.
```
Comment (nanjiangshu) [inline CONTRIBUTING.md:19]: Are we going to have a `release` branch? If true, the production web site might be better based on the `release` branch, if not, shouldn't it be based on the latest release tag?

**Review by nanjiangshu — COMMENTED** [review]
Looks great to me! I have just one question regarding how the production website should  be based.

File: CONTRIBUTING.md (hunk 2 above)
Comment (inghylt) [reply CONTRIBUTING.md:19]: I think we should discuss this during the next standup!

**Review by MalinAhlberg — APPROVED** [review]
Nice :star:

**Pull request merged**